cmake/modules/FindAbseil.cmake

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT SUBMIT, would break on CMake versions.
+# FIXME: BUILD_BYPRODUCTS requires CMake 3.2.
+cmake_minimum_required(VERSION 3.2)
+
+if(NOT TARGET standalone_absl)
+  include(${CMAKE_ROOT}/Modules/ExternalProject.cmake)
+  set(ABSEIL_PROJECT_NAME abseil)
+  set(ABSEIL_PROJECT_SRC_DIR
+    ${CMAKE_CURRENT_BINARY_DIR}/${ABSEIL_PROJECT_NAME}/src/${ABSEIL_PROJECT_NAME})
+  set(ABSEIL_PROJECT_BUILD_DIR
+    ${CMAKE_CURRENT_BINARY_DIR}/${ABSEIL_PROJECT_NAME}/src/${ABSEIL_PROJECT_NAME}-build)
+  set(ABSEIL_INCLUDE_DIRS ${ABSEIL_PROJECT_SRC_DIR})
+  set(ABSEIL_LIBRARY_PATH
+    "${ABSEIL_PROJECT_BUILD_DIR}/absl/synchronization/${CMAKE_STATIC_LIBRARY_PREFIX}absl_synchronization${CMAKE_STATIC_LIBRARY_SUFFIX}")
+  set(dependent_libraries
```

Comment:
style: rename to ABSEIL_DEPENDENT_LIBRARIES

Reply:
done